pom.xml

```diff
@@ -494,7 +494,7 @@
         <repository>
             <id>osgeo</id>
             <name>Open Source Geospatial Foundation Repository</name>
-            <url>https://download.osgeo.org/webdav/geotools/</url>
+            <url>https://repo.osgeo.org/repository/geotools-releases/</url>
```

Comment:
Why is this not `<url>https://repo.osgeo.org/repository/release/</url>` like the documentation says? I tryed both and for some reason both seems to work.

Reply:
PS! I did remove the `org.geotools` from my local m2 cache.

Reply:
You're right, is should be better, i've modified the url